inference-engine/thirdparty/clDNN/api/cldnn/primitives/convolution.hpp

```diff
@@ -774,7 +764,12 @@ struct convolution : public primitive_base<convolution> {
     /// @param padding_below Defines a padding added to input image on right (x axis) and bottom (y axis).
     tensor padding_below;
     /// @param deformable_mode.
-    bool deformable_mode;
+    bool deformable_mode {false};
+    /// if bilinear_interpolation_pad is true and the sampling location is within one pixel outside of the feature map boundary,
+    /// then bilinear interpolation is performed on the zero padded feature map.
+    /// If bilinear_interpolation_pad is false and the sampling location is within one pixel outside of the feature map boundary,
+    /// then the sampling location shifts to the inner boundary of the feature map.
+    bool bilinear_interpolation_pad;
```

Comment:
Seems like this field may be uninitialized in some c-tors.
`bool bilinear_interpolation_pad = false`

Reply:
Added `bilinear_interpolation_pad` initialization